CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @vanessa-cooper @carl-rattmann
```

Comment:
Do I need to remove the  '*' symbol from the start?
I don't know why this error is coming.